index.html

```diff
@@ -3206,10 +3405,13 @@ <h3 id="Attacking">Attacking</h3>
          </p>
          <p>
             To hit the target you must roll equal to or higher than their armour class (AC).
+            If you use any ability which requires an attack roll you still hit if you roll lower than
```

Comment:
Generally like this - does this affect NPCs?  I'd assume it can, particularly for bosses.  This would also affect the balance of auto-hit abilities but I'm not entirely sure how yet